react/src/App.tsx

```diff
@@ -3,13 +3,40 @@ import Header from "./components/Header";
 import ModeSwitcher from "./components/ModeSwitcher";
 
 function App() {
-  const [ mode, setMode] = useState<'light' | 'dark'>('light')
+  const [mode, setMode] = useState<"light" | "dark">("light");
   return (
     <div className={`App sdds-mode-${mode}`}>
-      <Header/>
-      <ModeSwitcher  mode={mode} setMode={setMode} />
+      <Header />
+      <ModeSwitcher mode={mode} setMode={setMode} />
       <main>
         {/* COMPONENTS GO HERE */}
+        <div>
+          <div className="sdds-headline-02">Accordion</div>
+          <sdds-accordion>
+            <sdds-accordion-item header="First item">
+              This is the panel, which contains associated information with the
+              header. Usually it contains text, set in the same size as the
+              header. Lorem ipsum doler sit amet.
+            </sdds-accordion-item>
+            <sdds-accordion-item header="Second item">
+              This is the panel, which contains associated information with the
+              header. Usually it contains text, set in the same size as the
+              header. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+              Duis laoreet vestibulum fermentum.
+            </sdds-accordion-item>
+          </sdds-accordion>
+        </div>
+        <div>
+          <div className="sdds-headline-02">Banner</div>
```

Comment:
```suggestion
          <div className="sdds-headline-02 sdds-u-pb1">Banner</div>
```
Just to get some spacing...